.github/workflows/update-holonix-version.yml

```
@@ -19,27 +19,39 @@ concurrency:
 
 jobs:
   update-holonix-version:
-    runs-on: [ self-hosted, multi-arch ]
+    runs-on: "ubuntu-latest"
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v4
+        with:
+          ref: develop
+      - uses: cachix/install-nix-action@v22
+      - name: Get tags
+        run: git fetch --quiet --tags origin
```

IMO more info is good on CI

```suggestion
        run: git fetch --tags origin
```

---

It prints like 2k lines of mostly useless information, if the script doesn't find the tags it's looking for then the PR should be rejected. I think that's okay. If we need to work out why a tag was missing, we know what commit hash the pipeline ran against and when it ran, that should be enough to trace the missing tag.